pageserver/src/tenant/secondary.rs

```diff
@@ -66,7 +66,7 @@ struct CommandRequest<T> {
 }
 
 struct CommandResponse {
-    result: anyhow::Result<()>,
+    result: Result<(), ApiError>,
```

Comment:
Alternatively, we could make the scheduler generic over the error type, but this seems fine for now.